scripts/package-windows-images

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this file at all if we're not going to do Windows on this branch?

Reply:
ya i thought i deleted that one, i got the others... good spot